demo/admin/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "1.0.0",
     "private": true,
     "scripts": {
-        "admin-generator": "rimraf 'src/*/generated' && comet-admin-generator generate",
+        "admin-generator": "rimraf -g 'src/**/generated' && $npm_execpath gql:types && comet-admin-generator generate && $npm_execpath gql:types",
```

Comment:
I'd use `--glob` here, then devs don't have to look up rimraf's docs (could be global as well)